### PR TITLE
BiNormalisationSPECT: takes image duration from exam_info

### PR DESCRIPTION
**Pull request by danieldeidda**

### CODE REVIEW

File: src/recon_buildblock/BinNormalisationSPECT.cxx [hunk 1]
```diff
@@ -106,6 +105,8 @@ post_processing()
   norm_proj_data_info_ptr=ProjData::read_from_file(projdata_filename);
   max_tang=norm_proj_data_info_ptr->get_max_tangential_pos_num();
   
+  this->view_time_interval=get_exam_info_sptr()->get_time_frame_definitions().get_duration(0)/num_views;
```
Comment (KrisThielemans) [inline src/recon_buildblock/BinNormalisationSPECT.cxx:108]: probably best to have a check here on number of time frames()==1

File: src/recon_buildblock/BinNormalisationSPECT.cxx [hunk 2]
```diff
@@ -80,16 +80,15 @@ BinNormalisationSPECT::
...
-  this->parser.add_key("use_uniformity_factors", &this->_use_uniformity_factors);
-  this->parser.add_key("folder_prefix", &this->folder_prefix);
-  this->parser.add_key("rel_angle", &this->rel_angle);
-  this->parser.add_key("half_life", &this->half_life); //TODO read this from the database according to isotope name
-  this->parser.add_key("view_time_interval", &this->view_time_interval);
+  this->parser.add_key("uniformity filename", &this->uniformity_filename);
+  this->parser.add_key("use detector efficiencies", &this->_use_detector_efficiencies);
+  this->parser.add_key("use uniformity factors", &this->_use_uniformity_factors);
+  this->parser.add_key("folder prefix", &this->folder_prefix);
+  this->parser.add_key("rel angle", &this->rel_angle);
```
[the remark this answers is not in the document]
Comment (danieldeidda) [reply src/recon_buildblock/BinNormalisationSPECT.cxx:87]: Angular step (degrees)?

File: src/recon_buildblock/BinNormalisationSPECT.cxx (hunk 1 above)
Comment (danieldeidda) [reply src/recon_buildblock/BinNormalisationSPECT.cxx:108]: something like:
if (false)
error(multiple image frame are not yet supported)

?

Comment (KrisThielemans) [reply src/recon_buildblock/BinNormalisationSPECT.cxx:110]: also a check `f (this->get_exam_info_sptr()->get_time_frame_definitions().get_num_frames()==0) error()`

**Pull request merged**